### DISCUSSION TITLE
unmet peer eslint@^8.56.0: found 9.13.0 / can't install latest vitest version with pnpm

### DISCUSSION THREAD

**Discussion by kolorfilm** [vitest-dev/vitest#6754] [Q&A]
### Describe the bug

WARN  Issues with peer dependencies found
.
└─┬ eslint-plugin-vitest 0.5.4
  └─┬ @typescript-eslint/utils 7.18.0
    └── ✕ unmet peer eslint@^8.56.0: found 9.13.0

### Reproduction

unmet peer eslint@^8.56.0: found 9.13.0

### System Info

```shell
System:
    OS: macOS 15.0.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 64.28 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.18.0 - /opt/homebrew/opt/node@20/bin/node
    npm: 10.8.2 - /opt/homebrew/opt/node@20/bin/npm
    pnpm: 9.12.2 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 130.0.6723.59
    Safari: 18.0.1
  npmPackages:
    @vitejs/plugin-react-swc: ^3.7.1 => 3.7.1 
    @vitest/eslint-plugin: ^1.1.7 => 1.1.7 
    vite: ^5.4.9 => 5.4.9 
    vitest: ^2.1.3 => 2.1.3
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hi-ogawa** [comment]
Probably you have old `eslint-plugin-vitest`. Can you try upgrade?
Their repo is https://github.com/vitest-dev/eslint-plugin-vitest/, so you should ask it there for further question.

**Comment by kolorfilm** [comment]
Hi there and thank you for your response,

it seems with pnpm it installs max v0.5.4. With pnpm I can't install the latest > v1 version.

<img width="1111" alt="Bildschirmfoto 2024-10-21 um 12 35 16" src="https://github.com/user-attachments/assets/003f6434-9c6c-46e3-b401-d1cbf4196d39">

**Comment by kolorfilm** [comment]
It seems on npm the latest version is 0.5.4 according to the versions page: https://www.npmjs.com/package/eslint-plugin-vitest?activeTab=versions

**Comment by kolorfilm** [comment]
Ok I found the solution. I have to install the eslint page differently. See the README here: https://github.com/vitest-dev/eslint-plugin-vitest

It works now.
